roles/openshift_logging/templates/kibana.j2

```diff
@@ -34,9 +34,9 @@ spec:
           imagePullPolicy: Always
           env:
             - name: "ES_HOST"
-              value: "{{es_host}}"
```

Comment:
we should keep this es_host since it can change depending on if this is deployed as part of ops or non-ops